### DISCUSSION TITLE
Workflow para zonas de roble-encinar 1º cobertura

### DISCUSSION THREAD

**Discussion by manudona92** [manudona92/MapantES#63] [General]
Las zonas de vegetación densa de roble y encinar salen con mucho ruido en algunos lotes de la 1º Cobertura (Extremadura N,  Aragón N y S, CyL NE, etc.).

Una forma de quitarlo es reconstruyendo la clasificación de suelo, con las ground. El workflow sería el siguiente

1) Convertir todo a clasificación clase 0 y quitar las de clase 6, 7, y 12 (edificios, ruido y solape). Creo Agustín que tu ya haces las2las, sería añadir esos dos comandos (drop classification y set classification)

las2las -i  -drop_classification 6 7 12 -set_classification 0  -olaz

2) Clasificar nube de puntos en suelo y no suelo con lasground_new

lasground_new -i  -olaz

Lasground_new por defecto usa los parámetros que necesitamos en este caso (captura de pantalla desde la interfaz)
![image](https://user-images.githubusercontent.com/76484741/117363540-002e0500-aebd-11eb-9bfb-2fed183f3c01.png)

3) Dar altura a los objetos sobre el suelo con lasheight

lasheight -i  -classify_below 0.3 2 -classify_between 0.3 1 3 -classify_between 1 3 4 -classify_above 3 5 -olaz

-------

Ejemplo de algunos cambios
![image](https://user-images.githubusercontent.com/76484741/117364628-6d8e6580-aebe-11eb-9bec-9f6501320224.png)

**Comment by agustincaballero** [comment]
Está claro, tan sólo una cosa: si es posible, pon el comando completo lasground_new que habría que invocar. Aunque coja unos parámetros por defecto, es mejor poner todos los parámetros explícitamente, así nos aseguramos de que se ejecuta con los valores que queremos.

**Reply by manudona92** [reply]
Para el lasground_new  con todos los parámetros: 

lasground_new -i -step 25 -bulge 2 -spike 1 -down_spike 1 -offset 0.05 -olaz

**Comment by manudona92** [comment]
Expongo lotes que llevarán este procedimiento (hasta la fecha 20210514): 

**1_COBERTURA/RGB/2010_LOTE5_CyL_NE_RIOJA/Huso_30/**

- Procesar TODO con Madrid_2.ini 
- Manu creará polígono de las zonas a parchear
- Procesar polígonos afectados con el proceso de lasground_new y Madrid_2.ini

**1_COBERTURA/RGB/2014_CyL_SW/Huso_30/**

- Procesar TODO con Madrid_2.ini 
- Manu creará polígono de las zonas a parchear
- Procesar polígonos afectados con el proceso de lasground_new y Madrid_2.ini

**1_COBERTURA/RGB/2010_LOTE8_EXT_NORTE/Huso_30/**

- Procesar TODO con Extremadura_1.ini
- Manu creará polígono de las zonas a parchear
- Procesar polígonos afectados con el proceso de lasground_new y Madrid_2.ini

**1_COBERTURA/RGB/2010_LOTE2_ARAGON_SUR/Huso_30/**

- Procesar TODO con Madrid_2.ini  **YA HECHO**
- Manu creará polígono de las zonas a parchear **YA HECHO**
- Procesar polígonos afectados con el proceso de lasground_new y Madrid_2.ini

**1_COBERTURA/RGB/2012_NAVARRA/Huso_30/**

- Procesar SOLO ZONA SUR con lasground_new y Madrid_2.ini  / Manu enviará KML de la zona afectada

**1_COBERTURA/RGB/2010_LOTE4_CyL_NW/Huso_30/**

- Procesar SOLO ZONA SUR con Madrid_2.ini  / Manu enviará KML de la zona afectada
- procesar SOLO ZONA norte-oeste con lasground_new y Madrid_2.ini  / Manu enviará KML de la zona afectada

**1_COBERTURA/RGB/2012_ASTURIAS/Huso_30/**

- Procesar TODO con lasground_new y Asturias_1.ini
